ppafm/common.py

```diff
@@ -329,9 +336,9 @@ def get_simple_df_weight(n=10, dz=0.1):
 
 def Fz2df(
     F,
-    dz=0.1,
-    k0=params["kCantilever"],
-    f0=params["f0Cantilever"],
+    dz,
+    k0,
+    f0,
     amplitude=1.0,
     units=16.0217656,
```

Comment:
This probably does not need to be divided into multiple lines. The comma on the final line is forcing the formatter to keep it on multiple lines.

Reply:
done in https://github.com/Probe-Particle/ppafm/pull/275/commits/2e825103ecb25625fe6cd95fa3e614ccf274c855